wf/__init__.py

```diff
@@ -0,0 +1,121 @@
+"""
+A basic pipeline for quantification of genomic features from short read data implemented with Nextflow and wrapped inside the Latch SDK.
+"""
+
+import subprocess
+from pathlib import Path
+from typing import List
+from latch import medium_task, workflow
+from latch.resources.launch_plan import LaunchPlan
+from latch.types import LatchAuthor, LatchFile, LatchMetadata, LatchParameter, LatchDir
+
+
+@medium_task
+def rnaseq_task(
+    reads: List[LatchFile] = [
+        LatchFile("latch:///rnaseq-nf/ggal_gut_1.fq"),
+        LatchFile("latch:///rnaseq-nf/ggal_gut_2.fq"),
+    ],
+    transcriptome: LatchFile = LatchFile(
+        "latch:///rnaseq-nf/ggal_1_48850000_49020000.Ggal71.500bpflank.fa"
+    ),
+    outdir: LatchDir = LatchDir("latch:///rnaseq-nf/results"),
+) -> LatchDir:
+
+    local_outputdir = outdir.local_path
+    print(local_outputdir)
+    subprocess.run(["ls", local_outputdir])
+
+    Path(reads[0].local_path).rename("/root/read_1.fq")
+    Path(reads[1].local_path).rename("/root/read_2.fq")
+
+    files = "/root/read_{1,2}.fq"
+
+    _nf_cmd = [
```

Comment:
i'd name this `nf_cmd` instead of `_nf_cmd` to be consistent with the rest of your function